dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/src/Microsoft.ApplicationModel.Resources.vcxproj

```diff
@@ -188,15 +180,24 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </ProjectReference>
   </ItemGroup>
+  <PropertyGroup>
+    <Ixp-Arch Condition="'$(Platform)' == 'AnyCPU'">x86</Ixp-Arch>
```

Comment:
Can we call this something more generic than IXP?

Reply:
We don't want to accidentally collide with a built in name. do you have a recommendation?

Reply:
Hmmm....are we the first to consume this kind of package? What about Udk-Arch?